agents/unix/conf/base/conf_cpu.c

```diff
@@ -895,6 +899,83 @@ cpu_thread_grab(const char *name)
     return rc;
 }
 
+static te_errno
+memory_get(unsigned int gid, const char *oid, char *value)
+{
+  long n_pages, page_size;
+  uint64_t mem;
+
+  UNUSED(gid);
+  UNUSED(oid);
+
+  errno = 0;
+
+#ifdef _SC_PHYS_PAGES
+  n_pages = sysconf(_SC_PHYS_PAGES);
+#else
+  return 0;
```

Comment:
This might produce a warning on unused `page_size` and a warning about dead code that follows.